deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -331,6 +331,13 @@ cephClusterSpec:
       requests:
         cpu: "500m"
         memory: "100Mi"
+    exporter:
+      limits:
+        cpu: "300m"
+        memory: "512Mi"
```

Comment:
Per discussion in huddle, how much memory are you seeing used in practice? Even 512Mi seems like a high limit.